### DISCUSSION TITLE
400S Turn on to a fan setting or smart mode

### DISCUSSION THREAD

**Discussion by RagnarSTS** [RobertD502/home-assistant-iocare#55] [Q&A]
I will admit that I might just be looking at this incorrectly so I might be silly.  If I am clearly in error, please feel free to tell me how to do this.  So I've had a 400S for a couple years, love it, added a second one this last week hence my digging into things more.  

I really want to set up some automations to trigger something like 'midnight; turn on 400S high fan speed' and '0700 turn on smart mode'.  Extra credit for a I burned something in the kitchen fire and the air quality index is bad, run full bore until it gets below 40 and then run for another 30 minutes before you drop to smart mode.
Sorry...the built in smart mode is just a little too forgiving. 

At any rate, I cannot figure out how to automate turning it on to a specific fan setting or smart mode.  When I make a card and put it on the overview it has all those functions.  I can on/off, and additionally set fan to whatever speed or whatever smart sensitivity. 
The automations only give me things like 'next sensitivity' .... this is kind of useless.  I don't want to scroll through things I just want to set it for reals becuase inevitably somebody will touch it and then scrolling through won't be correct anymore. 

How do I approach this?

**Comment by AbsoluteFeaturitis** [comment]
Start with a script.

1. **Under Automation**
![image](https://github.com/user-attachments/assets/5a512e9d-806b-40dc-b8ef-37b2dfed7989)

2. **Select Scripts** (from the top if you're on PC web browser)
![image](https://github.com/user-attachments/assets/e99933f1-5d93-4d46-bf4f-f31bcb3aea88)

3. **Select Create Script**
![image](https://github.com/user-attachments/assets/82616975-e4c1-4a00-b1a4-8dee99c77caf)

4. **Select Create New Script**
![image](https://github.com/user-attachments/assets/fa025d4a-2d0a-45da-86c2-6b5752f8c7ea)

5. **Select Add Action**
![image](https://github.com/user-attachments/assets/f006f718-da3a-4ce1-b4ce-ee33a8dd1e94)

6. **Select Fan**
![image](https://github.com/user-attachments/assets/81fec400-5846-4e1f-9fda-df53f499b9a3)

7. **Select Set Speed**
![image](https://github.com/user-attachments/assets/95b20a16-48b7-4545-983f-7e16183550e9)

8. **Select Choose Entity**
![image](https://github.com/user-attachments/assets/b55f3e29-d761-4dc3-8429-9437a59a7ea2)

9. **Choose Your Coway Device Name**
![image](https://github.com/user-attachments/assets/70322a6c-41b6-45f1-a6b4-11ff77c8756d)

10. **Set the Fan Speed between 0-100**
![image](https://github.com/user-attachments/assets/a0a3e44a-9003-46ec-a33d-18d930f5e470)

11. **Select Save Script**
![image](https://github.com/user-attachments/assets/5aca3a56-950a-4e3f-a06e-d648020ae2f1)

12. **Name and Save Your Script**
![image](https://github.com/user-attachments/assets/d90f1ce0-248e-4176-b89b-a7e4899aac39)

Then Create an Automation.

1. **Under Automation**
![image](https://github.com/user-attachments/assets/5a512e9d-806b-40dc-b8ef-37b2dfed7989)

2. **Select Automation** (from the top if you're on PC web browser)
![image](https://github.com/user-attachments/assets/aa83222f-1e9a-421a-9daf-137d5f86777d)

3. **Select Create Automation**
![image](https://github.com/user-attachments/assets/cebcb7ae-ca4c-4778-9eda-7e72591a85d1)

4. **Select Create New Automation from Scratch**
![image](https://github.com/user-attachments/assets/ed27d7a9-0835-4299-964e-a2068df3c16f)

5. **Select Add Trigger** (Add whatever trigger you like and save it)
![image](https://github.com/user-attachments/assets/acf520b8-2816-4de9-a067-5eff562680d1)

6. **Select "Add Action"**
![image](https://github.com/user-attachments/assets/e90b8760-e0d6-4e30-bb3c-1302406ec56a)

7. **Type Script in the Search Box and Select the Script You Created** 
![image](https://github.com/user-attachments/assets/e6922c5b-ad3d-4610-a242-bf9e9dff2c82)

8. **Select Save**
![image](https://github.com/user-attachments/assets/45592640-3016-46b8-bf77-24001de38cef)

Finally Test Your Automation.

Any other automation involving your Coway would follow a similar procedure.  

Respect to the Coway User Group and the Honorable RobertD502.
